service/src/test/java/uk/gov/hmcts/reform/fpl/service/HearingBookingServiceTest.java

```diff
@@ -132,29 +123,79 @@ private DynamicListElement createDynamicElement(UUID code) {
         }
     }
 
+    @Nested
+    class GetPastHearings {
+
+        @Test
+        void shouldReturnEmptyListWhenNoHearingsHaveBeenCreated() {
+            assertThat(service.getPastHearings(emptyList())).isEqualTo(emptyList());
```

Comment:
minor, isEqualTo(emptyList()) can be replaced with .isEmpty()

Reply:
yep, silly of me